src/additionalFunctions/index.ts

```diff
@@ -0,0 +1,10 @@
+import { User } from '../types/user';
+import { Comment } from '../types/comment';
+
+export function getUsers(userId:number, users: User[]) : User {
+  return users.filter(u => u.id === userId)[0];
+}
```

Comment:
1. You also could use method find;
2. What does variable 'u' mean? You should use more understandable names for variables.

Reply:
+1
Fix it